### PR TITLE
add mobile styles

### PR DESCRIPTION
**Pull request by YuriyKrasnobokiy**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
```
Comment (stas-yushkov) [inline index.css:28]: duplicates line 9, 10

File: index.css [hunk 2]
```diff
@@ -1,3 +1,45 @@
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul {
```
Comment (stas-yushkov) [inline index.css:8]: add ol here

File: index.css [hunk 3]
```diff
@@ -1,3 +1,43 @@
+:root {
+  --accent-color: rgb(67, 155, 245);
```
Comment (stas-yushkov) [inline index.css:2]: add all colors here

File: index.css [hunk 4]
```diff
@@ -106,25 +193,41 @@ h1 {
...
+    text-align: center;
+  }
+}
+
 /* BARS */
 
-ol {
-  list-style: none;
-  padding: 0 10px;
+/* @media screen and (max-width: 1023px) { */
```
Comment (stas-yushkov) [inline index.css:205]: dell all comments

File: index.html [hunk 5]
```diff
@@ -231,10 +232,13 @@ <h1>Розрахунок міцності коктейлів (або пийте
...
-            <img src="./images/алкоШот.png" alt="алкаШот" width="150" />
+            <img src="./images/alco-shoot.png" alt="Alco Shoot" width="150" />
           </div>
           <p class="results-title">Міцність коктейлю:</p>
-          <p class="results-desc JS-result">
+          <p class="results-desc results-desc-number JS-result-number">
+            ...
+          </p>
+          <p class="results-desc results-desc-text JS-result-text">
             ... (тицяй велику кнопку нижче) ...
```
Comment (stas-yushkov) [inline index.html:242]: del round brackets

File: index.css [hunk 6]
```diff
@@ -1,6 +1,52 @@
...
+}
+
+a {
+  text-decoration: none;
+  color: var(--current-color);
+  outline: none;
+}
+
 body {
   background-color: rgb(34, 39, 46);
```
Comment (stas-yushkov) [inline index.css:48]: add to :root

File: index.css [hunk 7]
```diff
@@ -13,51 +59,89 @@ body {
...
+  .hero-text {
+    font-weight: normal;
+    font-size: 16px;
+  }
 }
 
+
 .hero {
   padding: 250px 40px 70px 40px;
   background-image: linear-gradient(
```
Comment (stas-yushkov) [inline index.css:147]: add colors to :root

File: index.css (hunk 3 above)
Comment (stas-yushkov) [reply index.css:2]: not all yet

File: index.css [hunk 8]
```diff
@@ -74,14 +158,42 @@ img {
...
+.hero-button:focus {
+  color: var(--accent-color);
 }
 
 h1 {
-  /* font-family: Roboto, sans-serif; */
   text-align: center;
-
   font-size: 40px;
   font-weight: bolder;
```
Comment (stas-yushkov) [inline index.css:198]: Please use numbers instead of 'bolder'
here and in 6 other places

Comment (stas-yushkov) [reply index.css:198]: here is ok 
`.description {
  text-align: center;
  line-height: 1.5;
  letter-spacing: 0.03em;

  font-size: 25px;
  **font-weight: 300;**

  margin-left: auto;
  margin-right: auto;
  padding: 0 0 30px;
  margin: 0;
}`

File: index.html [hunk 9]
```diff
@@ -72,7 +73,9 @@
           обчислень у вашій голові! Просто оберіть коктейль та (або) його
           інгредієнти і тицяйте на велику кнопку "розрахувати"
         </p>
-        <button type="button" class="hero-button">до калькулятору</button>
+        <div class="hero-button-container">
+          <a href="#calculator" class="hero-button">до калькулятору</a>
```
Comment (stas-yushkov) [inline index.html:77]: style this link like a button

File: index.css (hunk 8 above)
Comment (YuriyKrasnobokiy) [reply index.css:198]: why?

Comment (YuriyKrasnobokiy) [reply index.css:198]: done

File: index.css [hunk 10]
```diff
@@ -1,8 +1,59 @@
+:root {
+  --accent-color: rgb(67, 155, 245);
+  --current-color: #adbac2;
+
+  --body-bg-color: rgb(34, 39, 46);
+
+  --header-bg-color: rgba(45, 51, 59, 0.3);
+
+  --hero-bg-img-color: linear-gradient(rgba(47, 48, 58, 0.4), rgba(47, 48, 58, 0.4));
```
Comment (stas-yushkov) [inline index.css:9]: this is a gradient, not color. separate colors or rename variable

File: index.css [hunk 11]
```diff
@@ -13,57 +64,92 @@ body {
...
 }
 
+
 .hero {
   padding: 250px 40px 70px 40px;
-  background-image: linear-gradient(
-      rgba(47, 48, 58, 0.4),
-      rgba(47, 48, 58, 0.4)
-    ),
+  background-image: var(--hero-bg-img-color),
```
Comment (stas-yushkov) [inline index.css:152]: Even if the images are opaque and the color won't be displayed in normal circumstances, web developers should always specify a [background-color](https://developer.mozilla.org/en-US/docs/Web/CSS/background-color). If the images cannot be loaded—for instance, when the network is down—the background color will be used as a fallback.

Comment (stas-yushkov) [reply index.css:152]: just add bg color

File: index.css (hunk 10 above)
Comment (YuriyKrasnobokiy) [reply index.css:9]: change in mobile-menu branch

File: index.css (hunk 11 above)
Comment (YuriyKrasnobokiy) [reply index.css:152]: ?

File: index.css (hunk 8 above)
Comment (stas-yushkov) [reply index.css]: you can add these font-weight numbers to variables (:root), if you wish))

File: index.html (hunk 9 above)
Comment (stas-yushkov) [reply index.html:77]: https://prnt.sc/Q3M8NsWvdTAD

File: index.html [hunk 12]
```diff
@@ -84,7 +87,7 @@ <h1>Розрахунок міцності коктейлів (або пийте
         обчислень у вашій голові! Просто оберіть коктейль та (або) його
         інгредієнти і тицяйте на велику кнопку "розрахувати".
       </p>
-      <div class="bars-results">
+      <div  id="calculator" class="bars-results">
```
Comment (stas-yushkov) [inline index.html:90]:
```suggestion
      <div id="calculator" class="bars-results">
```

File: index.css [hunk 13]
```diff
@@ -13,57 +64,92 @@ body {
...
+  
+}
+
+@media screen and (max-width: 1023px) {
+  .header-container {
+    justify-content: flex-end;
+    height: 50px;
+
+  }
+}
```
Comment (stas-yushkov) [inline index.css:105]:
```suggestion
@media screen and (max-width: 1023px) {
  .header-container {
    justify-content: flex-end;
    height: 50px;
  }
}
```

File: index.css [hunk 14]
```diff
@@ -13,57 +64,92 @@ body {
...
-  max-width: 100%;
-  height: auto;
+@media screen and (max-width: 1023px) {
+  .hero-text {
+    font-weight: 400;
+    font-size: 16px;
+  }
 }
 
+
```
Comment (stas-yushkov) [inline index.css:149]:
```suggestion
```

File: index.css [hunk 15]
```diff
@@ -74,57 +160,104 @@ img {
...
   margin: 0;
-  padding: 30px 50px;
+  padding: 0 0 15px 0;
+  }
 }
 
-.progress-bars {
-  /* margin-right: 100px;
-  margin-left: auto; */
+
```
Comment (stas-yushkov) [inline index.css:256]:
```suggestion
```

File: index.css [hunk 16]
```diff
@@ -135,103 +268,180 @@ ol {
...
+    align-items: center;
+    justify-content: center;
+    font-size: 20px;
+    font-weight: 700;
+  }
+}
+
 .number {
-  /* font-family: sans-serif; */
+
```
Comment (stas-yushkov) [inline index.css:287]:
```suggestion
```

File: index.css [hunk 17]
```diff
@@ -135,103 +268,180 @@ ol {
...
   font-size: 25px;
-  font-weight: bolder;
+  font-weight: 700;
   align-items: center;
 
-  background: rgb(67, 155, 245);
+  background: var(--button-bg-color);
   border: 0px;
   border-radius: 10px;
+
```
Comment (stas-yushkov) [inline index.css:424]:
```suggestion
```

**Review by stas-yushkov — APPROVED** [review]

**Pull request merged**